frontend/apps/reader/modules/readerdogear.lua

```diff
@@ -71,7 +71,7 @@ function ReaderDogear:onReadSettings(config)
 end
 
 function ReaderDogear:onSetPageMargins(margins)
-    if self.ui.document.info.has_pages then
+    if not self.ui.rolling then
```

Comment:
self.ui.paging doesn't work here?

Reply:
It does, but I thought it would be more readable to just use `ui.rolling` here, as all this stuff mostly regards ReaderRolling - than mixing `not ui.paging` and `ui.rolling`.
(And I assume the cost of `not`, if any, is cheap enough :)

Reply:
Let me disagree and not to follow this.

Reply:
I'd glanced over this change but yeah, this doesn't strike me as a case where using `not` adds clarity.